core/src/test/java/apoc/load/LoadJsonTest.java

```diff
@@ -262,7 +263,66 @@ public void testLoadJsonByUrlInConfigFileWrongKey() throws Exception {
         }
     }
 
+    @Test
+    public void testLoadJsonWithAuth() throws Exception {
+        String userPass = "user:password";
+        String token = Util.encodeUserColonPassToBase64(userPass);
+        new MockServerClient("localhost", 1080)
+                .when(
+                        request()
+                                .withPath("/docs/search")
+                                .withHeader("Authorization", "Basic " + token)
+                                .withHeader("\"Content-type\", \"application/json\""),
+                        exactly(1))
+                .respond(
+                        response()
+                                .withStatusCode(200)
+                                .withHeaders(
+                                        new Header("Content-Type", "application/json; charset=utf-8"),
+                                        new Header("Cache-Control", "private, max-age=1000"))
+                                .withBody("{ result: 'message' }")
```

Comment:
fai così salva il messaggio in una hashmap, dopo di che qui la metti serializzata con jackson e a linea 290 invece controlli che le mappe di input e output siano uguali